adsmp/tests/test_solr_updater.py

```diff
@@ -663,6 +663,33 @@ def test_extract_data_pipeline(self):
             d["ned_object_facet_hier"],
         )
 
+        nonbib = {
+            "gpn": ["Moon/Crater/Langrenus/3273"]
+        }
+        d = solr_updater.extract_data_pipeline(nonbib, None)
+        self.assertEqual(["3273"], d["gpn_id"])
+        self.assertEqual(
+            ['0/Moon', '1/Moon/Crater', '2/Moon/Crater/Langrenus'],
+            d["gpn_facet_hier_3level"],
+        )
+        self.assertEqual(
+            ['0/Moon', '1/Moon/Crater/Langrenus'],
+            d["gpn_facet_hier_2level"],
+        )

```

Comment:
Add an assert for the `gpn` field itself, both for this example and for the next one